config/locales/account/en.yml

```diff
@@ -65,6 +65,15 @@ en:
         enter your email and password again.
       piv_cac: Sign in with your government employee ID
       tab_navigation: Account creation tabs
+    login_exit:
```

Comment:
Based on previous conversation, I might suggest using "cancel" terminology instead of "exit" for consistency